pkgs/development/tools/build-managers/buildbot/plugins.nix

```diff
@@ -1,85 +1,82 @@
 { stdenv, pythonPackages }:
 
-let
-  buildbot-pkg = pythonPackages.buildPythonPackage rec {
+# NOTE: buildbot plugins available as wheels
+
+{
+  www = pythonPackages.buildPythonPackage rec {
     name = "${pname}-${version}";
-    pname = "buildbot-pkg";
-    version = "0.9.7";
+    pname = "buildbot_www";
+    version = "0.9.9.post1";
+    format = "wheel";
 
     src = pythonPackages.fetchPypi {
-      inherit pname version;
-      sha256 = "0p351r10y42gwgxb2qg7xlsbhmnzdmqp6h4922l0yfii3pzmrdzv";
+      inherit pname version format;
+      sha256 = "0nsx4b1mrli89rikb1r91rbnyr8j8afpzplcnj0d1ivydbw6ldks";
     };
 
-    propagatedBuildInputs = with pythonPackages; [ setuptools ];
-
     meta = with stdenv.lib; {
       homepage = http://buildbot.net/;
-      description = "Buildbot Packaging Helper";
+      description = "Buildbot UI";
       maintainers = with maintainers; [ nand0p ryansydnor ];
       license = licenses.gpl2;
     };
   };
 
-in {
-  www = pythonPackages.buildPythonPackage rec {
+  console-view = pythonPackages.buildPythonPackage rec {
     name = "${pname}-${version}";
-    pname = "buildbot_www";
-    version = "0.9.7";
-
-    # NOTE: wheel is used due to buildbot circular dependency
+    pname = "buildbot_console_view";
+    version = "0.9.9.post1";
     format = "wheel";
 
     src = pythonPackages.fetchPypi {
       inherit pname version format;
-      sha256 = "1wf2spnilm0dkyw95vf57lca453sbly4r6ak3lxa8bpwhxb6lkdn";
+      sha256 = "1hs27zw0qwgnhrz9xsmhx4ahcz885pqxxxq6h51cxhdigwvsinc1";
     };
 
     meta = with stdenv.lib; {
       homepage = http://buildbot.net/;
-      description = "Buildbot UI";
+      description = "Buildbot Console View Plugin";
       maintainers = with maintainers; [ nand0p ryansydnor ];
       license = licenses.gpl2;
     };
   };
 
-  console-view = pythonPackages.buildPythonPackage rec {
+  waterfall-view = pythonPackages.buildPythonPackage rec {
     name = "${pname}-${version}";
-    pname = "buildbot-console-view";
-    version = "0.9.7";
+    pname = "buildbot_waterfall_view";
+    version = "0.9.9.post1";
+    format = "wheel";
 
     src = pythonPackages.fetchPypi {
-      inherit pname version;
-      sha256 = "1iv77886rbbn0wlzl5qiqc08rgbymxirqh3vmimqwsabbh7fhzkm";
+      inherit pname version format;
+      sha256 = "1jifcvvvfal38avizrxj10bmmi19s84yzva41q5v30sdnriim6ib";
     };
 
-    propagatedBuildInputs = with pythonPackages; [ buildbot-pkg ];
-
     meta = with stdenv.lib; {
       homepage = http://buildbot.net/;
-      description = "Buildbot Console View Plugin";
+      description = "Buildbot Waterfall View Plugin";
```

Comment:
Can we get a long description too, since https://pypi.python.org/pypi/buildbot-waterfall-view also doesn't bother to give one?